client/src/provides.ts

```diff
@@ -128,6 +131,8 @@ export interface Handler {
   groupAdd(): void;
   /* Put UI into group editing mode */
   groupEdit(id: AnnotationId | null): void;
+  /* Select next group in the list */
+  groupSelectNext(delta: number): void;
```

Comment:
I think I mentioned this elsewhere but I don't any actual usage of this provides function.